src/features/filter.ts

```diff
@@ -1,10 +1,38 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+type Select = { type: 'filter/SELECT', payload: Status };
+type Query = { type: 'filter/QUERY', payload: string };
+
+const setSelect = (value: Status): Select => (
+  { type: 'filter/SELECT', payload: value }
+);
```

Comment:
Consider renaming the function 'setSelect' to 'selectStatus' for better clarity.